pkg/pipeline/CiService.go

```diff
@@ -129,16 +136,48 @@ func (impl *CiServiceImpl) TriggerCiPipeline(trigger Trigger) (int, error) {
 		impl.Logger.Errorw("could not fetch ci config", "pipeline", trigger.PipelineId)
 		return 0, err
 	}
+	app, err := impl.appRepository.FindById(pipeline.AppId)
```

Comment:
this should be placed in a func. line 139-157 because we are doing this to extract only the namespace.